cmd/integration/integration.go

```diff
@@ -75,9 +75,9 @@ var (
 	// Limit the number of concurrent tests.
 	maxConcurrency int
 
-	longTestTimeout = time.Second * 300
+	longTestTimeout = time.Second * 500
```

Comment:
hmm - 300s doesn't seem to be to small timeout. I'm not convinced it can change anything...